src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  const basicDiscount = 20;
+  const extendedDiscount = 50;
+
+  switch (true) {
+    case days < 3:
```

Comment:
For numbers: `3` and `7`, you should create variables cause now they are 'magic numbers'